packages/angular/build/tools/baseline_browserslist/baseline_browserslist.bzl

```diff
@@ -0,0 +1,26 @@
+"""Generates a `browserslist` configuration from a Baseline date."""
```

Comment:
From what I’ve seen, we typically don’t create nested tools directories. In my experience, other repos also follow this patter tools are placed in the root-level tools directory. Also, since this might be needed in `@schematics/angular`, it could make more sense to move it there instead.

Example: https://github.com/angular/angular-cli/tree/main/tools

//cc @devversion maybe this nested approach is now preferred

Reply:
Yeah, I went back and forth with whether to put this in `@angular/build` or in the greater workspace.

I think what I ultimately landed on was that the Baseline date and its generated `browserslist` are Angular-wide configurations not specific to any package (hence why `BASELINE_DATE` ended up in `/constants.bzl` now). So I think it follows to do the same for the tool. I like the idea of a tool-specific `package.json` file. Moved to `//tools/baseline_browserslist/...`.

I opted to just have each package run the tool themselves rather than find a single location for generated `.browserslist` and manually copy it into each package.